azure-pipelines.yml

```diff
@@ -14,139 +14,48 @@ pr:
     - '*'
 
 variables:
-  - name: is-runtime-release
-    value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/Runtime-v')]
-  - name: is-sdk-release
-    value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/SDK-v')]
-  - name: Codeql.Enabled
-    value: true
+- name: Codeql.Enabled
+  value: true
+
+parameters:
+- name: pools
+  type: object
+  default:
+  - name: NetCore-Public
+    vmImage: ubuntu-latest # The public PR pool schema is different from the 1ES one. We use 'vmImage' which is a default azure pipeline schema.
```

Comment:
I'd be surprised if this is working as intending. Meaning, I don't believe `ubuntu-latest` exists in the `NetCore-Public` pool. It might be falling back to the Microsoft-hosted pool (`Azure Pipelines`), but the logs for the PR build would show that. If you take an image from [here](https://helix.dot.net/#1ESHostedPoolImagesWestUS-rg-Public), that should work for ya. I'd recommend: `build.ubuntu.2204.amd64.open`

Reply:
Those pools dont seem to support this; hence the vm IMage. Name is not even used, its just there as a placeholder, which is kinda gross